dusty/commands/logs.py

```diff
@@ -1,9 +1,12 @@
 from ..systems.docker import get_dusty_containers
 from .utils import exec_docker
 
-def tail_container_logs(app_or_service_name, lines=None):
+def tail_container_logs(app_or_service_name, follow=False, lines=None):
     container = get_dusty_containers([app_or_service_name])[0]
-    if lines is None:
-        exec_docker('logs', '-f', container['Id'])
-    else:
-        exec_docker('logs', '-f', '--tail={}'.format(lines), container['Id'])
+    args = ('logs',)
```

Comment:
Agreed, much more idiomatic to use lists for this sort of things. Tuples are immutable. This `+=` assignment technically works but it's very strange.